cartographer_turtlebot2/src/flat_world_imu_node_main.cc

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2016 The Cartographer Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "ros/ros.h"
+#include "sensor_msgs/Imu.h"
+
+namespace {
+
+constexpr double kFakeGravity = 9.8;
+constexpr int kSubscriberQueueSize = 150;
+constexpr char kImuTopic[] = "imu";
+constexpr char kFlatWorldImuTopic[] = "flat_world_imu";
+
+}  // namespace
+
+int main(int argc, char** argv) {
+  ::ros::init(argc, argv, "flat_world_imu");
+
+  ros::Time last_time;
+  ::ros::NodeHandle node_handle("~");
```

Comment:
As discussed offline: Remove ~ here and elsewhere, and let users configure their namespaces.